File: benchmark/results/python-multilang/run-003/b-baseline/output/metaengine_demo/ordering/services/order_service.py
"""OrderService for the ordering domain."""
from typing import Optional

from metaengine_demo.ordering.aggregates.order import Order


class OrderService:
    """OrderService application service."""

    def create(self, input: Optional[Order]) -> Order:
        """Create a new order."""
        raise NotImplementedError("not implemented")

    def findById(self, id: str) -> Optional[Order]:
        """Find an order by id."""
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[Order]:
        """List orders up to a limit."""
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        """Delete an order by id."""
        raise NotImplementedError("not implemented")

File: benchmark/results/java-multilang/run-005/b-baseline/output/com/metaengine/demo/ordering/services/OrderService.java
package com.metaengine.demo.ordering.services;

import com.metaengine.demo.ordering.aggregates.Order;
import java.util.List;

/** OrderService service for the ordering domain. */
public class OrderService {

    /** Create a new Order. */
    public Order create(Order input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an Order by id. May return null. */
    public Order findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List orders up to the given limit. */
    public List<Order> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an Order by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
